src/pages/UserPage/UserPage.jsx

```diff
@@ -135,41 +258,52 @@ const UserPage = () => {
               <RiEdit2Fill onClick={handleEditClick}  className='userPage__form-icon'/>
               {errors.password && <p className='restaurantAdminView__error'>{errors.password}</p>}
             </span>
-            <label className='userPage__form-label' htmlFor='userPhoneNumber'>Phone Number</label>
-            <span>
-              <input
-                className='userPage__form-input'
-                type='number'
-                id='userPhoneNumber'
-                name='phoneNumber'
-                disabled={!isEditable}
-                onChange={handleInputChange}
-                placeholder={user.phoneNumber}
-                value={formData.phoneNumber}
-              />
-              <RiEdit2Fill onClick={handleEditClick}  className='userPage__form-icon'/>
-              {errors.phoneNumber && <p className='restaurantAdminView__error'>{errors.phoneNumber}</p>}
-            </span>
-            <label className='userPage__form-label' htmlFor='userAddress'>Address</label>
-            <span>
-              <input
-                className='userPage__form-input'
-                type='text'
-                name='address'
-                id='userAddress'
-                disabled={!isEditable}
-                onChange={handleInputChange}
-                placeholder={user.address}
-                value={formData.address}
-              />
-              <RiEdit2Fill onClick={handleEditClick}  className='userPage__form-icon'/>
-              {errors.address && <p className='restaurantAdminView__error'>{errors.address}</p>}
-            </span>
-            <label className='userPage__form-label' htmlFor='userCity'>City</label>
+            {user.phone_numbers && user.phone_numbers.map((phoneNumber, index) => (
+              <div key={phoneNumber.id_user_phone_number}>
+                <label className='userPage__form-label' htmlFor='userPhoneNumber'>{languageSelector(language, 'signInPhone')} {index + 1}</label>
```

Comment:
consider not using the map index, why not using id entity item or uuid?

Reply:
So I'm using the index to name the input and display the number of the address or phone number, if I were to change it to id entity or uuid, would I still be able to do that?

Note that the key is the id that the item brings from the back end.

Reply:
ok, thx for clarifying, let's move this to resolved.